public/js/doctors.js

```diff
@@ -0,0 +1,11 @@
+const hamburger = document.getElementById('hamburger');
```

Comment:
Remove this file as it is the same as hamburger.js file and rename `hamburger.js` to `hamburger_menu.js`